nixos/modules/system/boot/networkd.nix

```diff
@@ -535,6 +550,22 @@ let
     };
   };
 
+  routingPolicyRulesOptions = {
+    options = {
+      routingPolicyRuleConfig = mkOption {
+        default = { };
+        example = { routingPolicyRuleConfig = { Table = 10; IncomingInterface = "eth1"; } ;};
```

Comment:
Maybe add a dualstacked example by setting `Family=both`, because this rule would otherwise be IPv4-only.